content/components/action-menu.mdx

```diff
@@ -102,89 +128,235 @@ Action menu is used for actions, navigation, to display secondary options, or si
+    />
+    <Caption>Don't use crossmark icons for single or multi select items.</Caption>
   </Dont>
 </DoDontContainer>
 
+### Multi-level
+
+A menu with multiple levels is a common UI component used in desktop and mobile applications. It allows users to access nested subitems without cluttering the screen.
+
+<img
+  width="960"
+  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
+  src="https://user-images.githubusercontent.com/980622/232519615-d10c928d-5253-4c54-bc66-7fa1e0f85b08.png"
+/>
+
+It's important to understand that menus can't include additional form elements like inputs. That means if you want to provide a input that filters the subitems that you can't rely on this component.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232558685-0874d0dd-6d58-42d6-a19b-be19cdf32cf8.png"
+    />
+    <Caption>If you want to present filterable data do this through a dialog.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232547208-aa63f85b-cc9c-4fd2-8222-c26ac993c0cc.png"
+    />
+    <Caption>Don't add additional form elements inside a menu.</Caption>
+  </Dont>
+</DoDontContainer>
+
+Since you can't include form elements within a menu that also means that you can't combine a [select panel](/components/selectpanel) with a action menu.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234094256-abf3f1ad-2d90-48f9-84aa-a8e17cc64c47.png"
+    />
+    <Caption>If you want to present filterable data do this through a dialog.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234094263-399afdd0-b2e5-45dc-a218-db4a7dcdd9be.png"
+    />
+    <Caption>Don't mix select panels with action menus.</Caption>
+  </Dont>
+</DoDontContainer>
+
+When using submenus don't use the trailing visual to communicate selected values within their submenu.
+Leave this space for submenu indicators.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232562573-a687110d-06cb-4756-8fb3-6b7e05665dd4.png"
+    />
+    <Caption>Use trailing visuals for submenu indicators or keyboard shortcuts.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232562567-bb96a0cb-a796-4834-bf91-e3ddff86c312.png"
+    />
+    <Caption>Don't use trailing visual to communicate selected items in submenus.</Caption>
+  </Dont>
+</DoDontContainer>
+
+On smaller viewports the menu should always be repositioned to fit within the viewport. This could mean that a submenu overlaps it's parent menu.
```

Comment:
```suggestion
On smaller viewports, the menu should always be re-positioned to fit within the viewport. This could mean that a submenu overlaps it's parent menu.
```